meerk40t/gui/propertypanels/textproperty.py

```diff
@@ -9,6 +9,40 @@
 
 _ = wx.GetTranslation
 
+class PromptingComboBox(wx.ComboBox) :
+    def __init__(self, parent, choices=[], style=0, **par):
+        wx.ComboBox.__init__(self, parent, wx.ID_ANY, style=style|wx.CB_DROPDOWN, choices=choices, **par)
+        self.choices = choices
+        self.Bind(wx.EVT_TEXT, self.OnText)
+        self.Bind(wx.EVT_KEY_DOWN, self.OnPress)
+        self.ignoreEvtText = False
+        self.deleteKey = False
+
+    def OnPress(self, event):
+        if event.GetKeyCode() == 8:
+            self.deleteKey = True
+        event.Skip()
+
+    def OnText(self, event):
+        currentText = event.GetString()
+        if self.ignoreEvtText:
+            self.ignoreEvtText = False
+            return
+        if self.deleteKey:
+            self.deleteKey = False
+            if self.preFound:
```

Comment:
This looks like it should crash but does not appear to crash. Should have been set in Init. But, this code doesn't seem to invoke. Since self.preFound (other than camel case issues) should hit an error since it's not set already.

Reply:
Will look into it

Reply:
Normalized variable names and predeclared a variable. In my environment stuff is update properly. A different font needs to be confirmed a) by pressing enter b) selecting one via the combobox c) leaving the field